libpod/container_commit.go

```diff
@@ -125,23 +125,48 @@ func (c *Container) Commit(ctx context.Context, destImage string, options Contai
 	// Workdir
 	importBuilder.SetWorkDir(c.Spec().Process.Cwd)
 
+	genCmd := func(cmd string) []string {
```

Comment:
At some point we need to do regexes for all of this. Reviewing these complex parsing functions makes my head hurt sometimes.